testsuite/testcases/src/compatibility_test.rs

```diff
@@ -43,17 +43,8 @@ impl NetworkTest for SimpleValidatorUpgrade {
         let first_node = first_batch.pop().unwrap();
         let duration = Duration::from_secs(5);
 
-        println!("1. Downgrade all validators to older version");
-        // Ensure that all validators are running the older version of the software
-        let validators_to_downgrade = ctx
-            .swarm()
-            .validators()
-            .filter(|v| v.version() != old_version)
-            .map(|v| v.peer_id())
-            .collect::<Vec<_>>();
-        batch_update(ctx, &validators_to_downgrade, &old_version)?;
```

Comment:
We probably don't need to remove this section, assuming the batch_update function is a no-op when the version to update to is the same as the version of the validator. Just to be safe it might be good to leave this in.

Reply:
hmm, you are right, since I added initial version part for k8s swarm in this pr as well, k8s node does have initial version when swarm launch, and which can make this step to be no-op